ckanext/scheming/plugins.py

```diff
@@ -340,6 +350,7 @@ def _load_schema_module_path(url):
         return
     p = os.path.join(os.path.dirname(inspect.getfile(m)), file_name)
     if os.path.exists(p):
+        watch_file(p)
```

Comment:
We don't need anything conditional here to see if we're actually running under paster serve?